simulator/vpx/service_content.go

```diff
@@ -83,4 +83,5 @@ var ServiceContent = types.ServiceContent{
 	HealthUpdateManager:         &types.ManagedObjectReference{Type: "HealthUpdateManager", Value: "HealthUpdateManager"},
 	FailoverClusterConfigurator: &types.ManagedObjectReference{Type: "FailoverClusterConfigurator", Value: "FailoverClusterConfigurator"},
 	FailoverClusterManager:      &types.ManagedObjectReference{Type: "FailoverClusterManager", Value: "FailoverClusterManager"},
+	TenantManager:               &types.ManagedObjectReference{Type: "TenantTenantManager", Value: "TenantManager"},
```

Comment:
Thanks @nikhail !  The change looks good overall, but one issue is having this field set for client versions < vim 6.9.1, which may error when calling `RetrieveServiceContent` since this field isn't included in older vmodl versions.  Not an issue for Go clients, but Java and python will error.  We've punted on model versioning so far, but may need to consider it here.  The untested patch below may do the trick (we can also add helpers for future use).  Or a workaround would be to leave the field as nil by default in vcsim, then on the client side either set the field in tests or handle the case in the client code when `TenantManager` is nil.  

```diff
modified   simulator/service_instance.go
@@ -17,6 +17,7 @@ limitations under the License.
 package simulator
 
 import (
+	"strings"
 	"time"
 
 	"github.com/google/uuid"
@@ -69,10 +70,17 @@ func NewServiceInstance(ctx *Context, content types.ServiceContent, folder mo.Fo
 	return s
 }
 
-func (s *ServiceInstance) RetrieveServiceContent(*types.RetrieveServiceContent) soap.HasFault {
+func (s *ServiceInstance) RetrieveServiceContent(ctx *Context, _ *types.RetrieveServiceContent) soap.HasFault {
+	content := s.Content
+	action := ctx.req.Header.Get("SOAPAction")
+	version := strings.Split(action, "/")[1]
+	if version >= "6.9.1" {
+		content.TenantManager = &types.ManagedObjectReference{Type: "TenantTenantManager", Value: "TenantManager"}
+	}
+
 	return &methods.RetrieveServiceContentBody{
 		Res: &types.RetrieveServiceContentResponse{
-			Returnval: s.Content,
+			Returnval: content,
 		},
 	}
 }
```